internal/util/topology.go

```diff
@@ -52,7 +52,8 @@ func GetTopologyFromDomainLabels(domainLabels, nodeName, driverName string) (map
 
 	// size checks on domain label prefix
 	topologyPrefix := strings.ToLower("topology." + driverName)
-	if len(topologyPrefix) > 63 {
+	const lenLimit = 63
```

Comment:
Do we know where the length of a string with 63 chars comes from? I would guess it is some CSI constant. Maybe it can be used instead of hacking around the linter warning?

@ShyamsundarR do you know?

Reply:
There is no const in the spec for this field, we need to create an issue in CSI spec to add const so that we don't need to have our own const